### PR TITLE
add new Appium commands

### PR DESCRIPTION
**Pull request by christian-bromann**
## Proposed changes

While I was working on https://github.com/appium/appium/pull/13950 I found that we don't have support for some commands.

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: packages/wdio-protocols/protocols/appium.json [hunk 1]
```diff
@@ -1413,5 +1426,41 @@
         "description": "An object containing two fields: 'result', which is the return value of the script itself, and 'logs', which contains 3 inner fields, 'log', 'warn', and 'error', which hold an array of strings logged by console.log, console.warn, and console.error in the script's execution."
       }
     }
+  },
+  "/session/:sessionId/appium/events": {
+    "POST": {
+      "command": "getEvents",
+      "description": "Get events stored in appium server.",
+      "ref": "https://github.com/appium/appium-base-driver/pull/365",
```
Comment (erwinheitzman) [inline packages/wdio-protocols/protocols/appium.json]: Would rather point to `https://github.com/appium/appium/blob/master/docs/en/commands/session/events/get-events.md`

File: packages/wdio-protocols/protocols/appium.json [hunk 2]
```diff
@@ -1413,5 +1426,41 @@
...
+        "name": "result",
+        "description": "A JSON hash of events like `{'commands' => [{'cmd' => 123455, ....}], 'startTime' => 1572954894127, }`."
+      }
+    }
+  },
+  "/session/:sessionId/appium/log_event": {
+    "POST": {
+      "command": "logEvent",
+      "description": "Store a custom event.",
+      "ref": "https://github.com/appium/appium-base-driver/pull/364",
```
Comment (erwinheitzman) [inline packages/wdio-protocols/protocols/appium.json]: Would rather point to `https://github.com/appium/appium/blob/master/docs/en/commands/session/events/log-event.md`

File: packages/wdio-protocols/protocols/appium.json (hunk 1 above)
Comment (christian-bromann) [reply packages/wdio-protocols/protocols/appium.json]: no idea why I didn't find these links, good catch!

**Comment by christian-bromann** [comment]
@erwinheitzman updated links

**Pull request merged**